### ISSUE TITLE
Generic a11y issues

### ISSUE THREAD

**Issue by alexnm** [jsheroes/jsheroes.io#260]
We should grab some of the easy fixes from Lighthouse and just apply them to the website. Color contrast is in general ok, the only problem is with our combination of black/blue, but that's a longer discussion around branding and I would leave it as it is for now.

<img width="440" alt="screenshot 2018-11-30 at 10 55 25" src="https://user-images.githubusercontent.com/9945366/49282110-96f56c80-f48e-11e8-89a4-852ee774dc2c.png">

**Comment by alexnm** [comment]
For the <html> lang thing we should create a separate `pages/_document.js` file - https://github.com/zeit/next.js/issues/2256
